config/locales/client.en.yml

```
@@ -4151,7 +4151,7 @@ en:
         one: "Earned this badge %{count} time"
         other: "Earned this badge %{count} times"
       granted_on: "Granted %{date}"
-      others_count: "Others with this badge (%{count})"
+      others_count: "Granted to others %{count} times"
```

Since this has plural now you will need to add the `one:` and `other:` keys:


```suggestion
      others_count:
        one: "Granted to others %{count} time"
        other: "Granted to others %{count} times"
```